packages/plugins/graphql-plugin-subscriptions-amqp/README.md

```diff
@@ -42,6 +42,16 @@ To close the connection with RabbitMQ:
 await plugin.close();
 ```
 
+## Options
+
+The following options are available in the plugin.
+
+-   **connection**: [AMQP](https://www.npmjs.com/package/amqplib) connection options or amqp url (e.g. `amqp://localhost`).
+-   **exchange**: (optional) Queue exchange, defaults to `neo4j.graphql.subscriptions.fx`.
+-   **reconnectTimeout**: (optional) Timeout (in ms) between reconnection attempts. If not set, the plugin will not reconnect. Note that if the first connection fails, it will not attempt to reconnect.
```

Comment:
Is it not perhaps sensible to reconnect by default? With the library, you wouldn't expect it to stop working if the database restarts, so should we not implement the same behaviour in this plugin?